pkg/test/options.go

```diff
@@ -38,6 +38,7 @@ type OptionsFields struct {
 	LeaderElectionName      *string
 	LeaderElectionNamespace *string
 	MemoryLimit             *int64
+	CPURequests             *int64
```

Comment:
I'm wondering if we want to surface this as CPU requests or specifically surface this as the number of worker threads allocated to the scheduler. I see the argument to surface it this way for configuration complexity, but if this is a value we expect users may tune surfacing it directly makes more sense IMO.

Reply:
I don't think I want users to tune this quite yet (I mean they can obviously tune this by messing with this value) but I'd rather view it as an "implementation detail" at this point